junit-interface/src/main/java/munit/internal/junitinterface/EventDispatcher.java

```diff
@@ -133,10 +134,29 @@ public void testIgnored(Description desc) {
         desc,
         new InfoEvent(desc, Status.Ignored) {
           void logTo(RichLogger logger) {
+            StringBuilder builder = new StringBuilder(1024);
+            boolean isPending = false;
+            for (Annotation annotation : desc.getAnnotations()) {
+              if (annotation instanceof Tag) {
+                Tag tag = (Tag) annotation;
+                String kind = tag.getClass().getName();
+                if (kind.equals("munit.Tag") && tag.value().equals("Pending")) {
```

Comment:
Thanks for your review! The reason for lines 143-146 was for simplicity, to avoid adding lots of Java and JS interfaces in the `munit.internal.junitinterface` package. However, based on your feedback, I’ll push some changes to that interface.